src/migrations/20221810114644-add-suggest-changes-table.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+CREATE TABLE IF NOT EXISTS suggest_change_set (
+  id serial primary key,
+  environment varchar(255) NOT NULL,
+  state varchar(255) NOT NULL,
+  project varchar(255) NOT NULL,
+  changes text NOT NULL,
```

Comment:
Should this be a JSON B column?

Reply:
Or should change events be modeled as it's own table?

Reply:
Could definitely be it's own table, I think that makes more sense